src/utils/selectors.js

```diff
@@ -1,18 +1,14 @@
 // @flow
-import { flatten, map, pipe, prop, uniq } from 'ramda'
+import { flatten, map, prop, uniq } from 'ramda'
 
 interface IEvent {
   date: Date;
 }
 
 const viewTags = talk => (talk.tags ? talk.tags : [])
 
-export let eventTags = pipe(
```

Comment:
Interesting :fire:
But tt still seems to me that `pipe(/***/)` using is more informative. Maybe because `|>` operator is very new and rare for me.

Reply:
We should always strive for using native implementation of functions and methods etc. 
🚀🚀🚀🚀